submissions/laneleka/html-css-popup/lib/normalize.css

```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Not critical, but it makes sense to take the version without comments.

Reply:
Done!